test.js

```diff
@@ -1,15 +1,10 @@
 import test from 'ava'
-import {
-  space,
-  width,
-  fontSize,
-  util
-} from './src'
+import {space, width, fontSize, util} from './src'
 
 const theme = {
```

Comment:
Sorry, I was making tons of changes and formatted this with prettier. If you'd like me to revert stylistic changes, I can do that.